### PR TITLE
Keep Smartspacer's client SDK

### PR DESCRIPTION
**Pull request by Goooler**
This is a copy-paste from https://github.com/LawnchairLauncher/lawnchair/blame/13-dev/proguard.pro#L164-L165, it could be integrated into AAR as https://github.com/KieronQuinn/Smartspacer/blob/b08008d2521a6109fd52757d5b34696bbe1af745/sdk-client/build.gradle#L17

### CODE REVIEW

File: sdk-client/consumer-rules.pro
Comment (KieronQuinn) [inline sdk-client/consumer-rules.pro:1]: Are you sure this works? I originally had this setup and it wasn't excluding the viewbinding classes

Comment (Goooler) [reply sdk-client/consumer-rules.pro:1]:
```diff
OLD: sdk-client-release-b08008d.aar
NEW: sdk-client-release-526a102.aar

 AAR      │ old       │ new       │ diff
──────────┼───────────┼───────────┼───────
      jar │ 353.8 KiB │ 353.8 KiB │   0 B
 manifest │     224 B │     224 B │   0 B
      res │  78.6 KiB │  78.6 KiB │   0 B
    other │     7 KiB │   7.1 KiB │ +52 B
──────────┼───────────┼───────────┼───────
    total │ 439.7 KiB │ 439.7 KiB │ +52 B

 JAR     │ old  │ new  │ diff
─────────┼──────┼──────┼───────────
 classes │  201 │  201 │ 0 (+0 -0)
 methods │ 1441 │ 1441 │ 0 (+0 -0)
  fields │  716 │  716 │ 0 (+0 -0)

=================
====   AAR   ====
=================

 size │ diff  │ path
──────┼───────┼────────────────
 52 B │ +52 B │ ∆ proguard.txt
──────┼───────┼────────────────
 52 B │ +52 B │ (total)
```

**Comment by Goooler** [comment]
Please squash this PR when you are ready to merge.

**Comment by KieronQuinn** [comment]
Released as 1.0.6, thank you!

**Pull request merged**